### DISCUSSION TITLE
Unable to install YouTube revanced

### DISCUSSION THREAD

**Discussion by teb0rr0** [ReVanced/revanced-discussions#1076] [Question]
When proceeding with the patch, the application closes and continues like this every time it is reopened and the same procedure is performed.
![Screenshot_20240925_160756](https://github.com/user-attachments/assets/732559c6-c245-4c43-a587-63de6f8a5a3b)
![Screenshot_20240925_161105](https://github.com/user-attachments/assets/f775ba99-aaf9-4cb4-a7b6-81390f36aaef)

**Comment by LisoUseInAIKyrios** [comment]
Try deleting everything and starting over (delete Manager, MicroG, and all patches apps).

**Reply by teb0rr0** [reply]
I already did it and still. I still have the same problem. :(

**Reply by oSumAtrIX** [reply]
Send crashlogs using logcat by filtering for "AndroidRuntime". Check online how to use logcat

**Reply by teb0rr0** [reply]
I will share the logs

[revanced-manager_logcat_20240927092558697357.log](https://github.com/user-attachments/files/17166375/revanced-manager_logcat_20240927092558697357.log)

**Reply by oSumAtrIX** [reply]
What file manager are you using

**Reply by teb0rr0** [reply]
Google Files

**Reply by oSumAtrIX** [reply]
Try older versions of ReVanced Manager such as v1.19 and up
